server/models/assessRisksAndNeeds/riskSummary.ts

```diff
@@ -0,0 +1,10 @@
+export type RiskScore = 'LOW' | 'MEDIUM' | 'HIGH' | 'VERY_HIGH'
+
+export default interface RiskSummary {
+  whoIsAtRisk?: string | null
+  natureOfRisk?: string | null
+  riskImminence?: string | null
+  riskIncreaseFactors?: string | null
+  riskMitigationFactors?: string | null
+  riskInCommunity: Partial<Record<RiskScore, string[]>>
```

Comment:
do you think you have to cater for riskInCommunity being null? (given than all other fields are nullable), aslo perhaps annoyingly the string[] being nullable?

(not sure where the api is located)

Reply:
this is a `val riskInCommunity: Map<RiskLevel, List<String>>` in the API